sdk/state/state.go

```diff
@@ -79,13 +81,21 @@ func NewChannelFromSnapshot(c Config, s Snapshot) *Channel {
 	return channel
 }
 
+// ChannelAccount is an account that, with another channel account, form a
+// channel. Two participants are needed to form a channel, referred to as remote
+// and local.
```

Comment:
Suggested rewording here. I removed the local/remote, because this type has no knowledge of those concepts and so one could look at the type and struggle to identify where is local and remote captured in the fields that are there.
```suggestion
// ChannelAccount holds the details that a Channel tracks for a Stellar account. A Channel tracks the details of two Stellar accounts, one for each participant in the channel.
```